eslint-remote-tester.config.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { getPathIgnorePattern } = require('eslint-remote-tester-repositories');
```

Comment:
Should we use the `repositories` from `eslint-remote-tester-repositories` instead?

Reply:
Currently it seems that this setup runs tests against a small set of ESLint plugin repositories, as part of CI. This approach is good, and I think `eslint-plugin-unicorn` has similar setup. They also use a second smoke test workflow for a ~5 hour test run against randomly picked ~10k repositories. 

So which ever approach is ok. Do you want to run these tests quickly for every CI run, or schedule them to run once a week for large set of repositories. Or both?

Reply:
Thanks for explaining both approaches.

I figured we could start by running against a set of ESLint plugins, since those repos are of course more relevant and likely to trigger our rules than any other random repository. The current set is small so it's quick to run during CI.

If there was a desire to run on a very large set of repos but less frequently, I'm fine with that too.